Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/adapters/PlatformPagerAdapter.java

```diff
@@ -17,10 +16,11 @@
 {
 	private Context mContext;
 
-	private final static int[] TAB_ICONS = {
+	private final static int[] TAB_ICONS =
+	{
 			R.drawable.ic_gamecube,
 			R.drawable.ic_wii,
-			R.drawable.ic_folder// wiiware TODO Have an icon here.
+			R.drawable.ic_folder // Wiiware TODO Have an icon here.
```

Comment:
If you are going to fix the capitalization for "WiiWare" the second W should be uppercase as well.